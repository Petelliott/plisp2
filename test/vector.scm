(println #(1 2 3 (+ 1 2)))
(println (vector 1 2 3 (+ 1 2)))

(define v #(1 2 3 4 5 6 7 8))

(println (vector-length v))
(println (vector-ref v 0) (vector-ref v 1) (vector-ref v 7))

(vector-set! v 3 99)
(println (vector-ref v 3))

(println (vector-append #() #()) (vector-append #(1 2) #(3 4)))
(println (make-vector 4 7))

(println (list->vector '())
         (list->vector '(1 2 3)))
