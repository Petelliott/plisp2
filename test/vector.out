#(1 2 3 (+ 1 2))
#(1 2 3 3)
8
1 2 8
99
#() #(1 2 3 4)
#(7 7 7 7)
#() #(1 2 3)
